opencti-platform/opencti-graphql/src/migrations/1708523502214-add-platform-data-sharing-max-markings.js

```diff
@@ -0,0 +1,18 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { getSettings, settingsEditField } from '../domain/settings';
+import { logApp } from '../config/conf';
+
+export const up = async (next) => {
```

Comment:
this migration file name need to be updated, its timestamp is lower than 1711444304636-create-index-deleted-objects.js which is already present in master